examples/platforms/efr32/logging.c

```diff
@@ -33,12 +33,14 @@
  */
 
 #include <openthread/config.h>
+#include <openthread-core-config.h>
 
 #include <openthread/cli.h>
 #include <openthread/platform/logging.h>
 
 #include "utils/code_utils.h"
 
+#if (OPENTHREAD_CONFIG_LOG_OUTPUT ==  OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED)
 void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat, ...)
 {
 #if OPENTHREAD_ENABLE_CLI_LOGGING
```

Comment:
This is as it was and I did not want to modify that code - I assume this might be something unique to the NXP tool chain or stuff provided by NXP external to openthread

Also see the note about the NCP and CLI where otPlatLog()  is called (comments from @abtink )